style.css

```diff
@@ -1,270 +1,563 @@
+  display: grid;
+  grid-template-columns: 1fr;
+  width: 94%;
+  margin-left: 3%;
+  border-radius: 10px;
+  background-color: #fff;
+  padding: 16px;
+}
 
-  /* Neutrals/N800 */
+.works-container-image {
+  width: 100%;
+}
+
+.must-ignor2 {
+  display: none;
+}
 
+.primarytext-title {
+  font-size: 38px;
+  margin-block: 15px;
   color: #172b4d;
 }
 
-#bg-work1 {
-  background-repeat: no-repeat;
-  box-sizing: border-box;
-  display: grid;
-  flex-direction: column;
-  align-items: center;
-  padding: 16px;
-  gap: 12px;
-  width: 327px;
-  height: 680px;
-  background-color: #fff;
-  opacity: 20px;
-  border: 1px solid #dfe1e6;
-  border-radius: 16px;
-  margin-top: 170px;
+.frame {
+  display: flex;
   justify-content: space-between;
+  align-items: center;
 }
 
-.work-container {
-  display: grid;
+.stack-year .title {
+  color: #344563;
 }
 
-.container {
-  display: grid;
-  width: 375px;
+.lang-year {
+  color: #7a869a;
 }
 
-.about {
-  margin-top: 70px;
+.works-container-text {
+  margin-block: 10px;
+  color: #344563;
+  font-size: 15px;
+}
+
+.btns {
+  margin-block: 16px;
+  display: flex;
+  gap: 2px;
+  text-transform: lowercase;
 }
 
-.work-content h2 {
+.primarytext1-1 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  margin-top: -8px;
   width: 295px;
-  height: 36px;
-  font-family: "Poppins", sans-serif;
-  font-weight: 900;
-  font-size: 32px;
-  line-height: 44px;
-  color: #172b4d;
+  height: 124px;
+  background: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
 }
 
-#frame2 {
-  width: 100%;
+ul.primarytext3 {
+  margin-block: 16px;
+  display: flex;
+  align-items: center;
 }
 
-#frame2 h2 {
-  width: 88px;
-  height: 16px;
-  font-family: "Poppins", sans-serif;
+.canopy {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.num-text {
+  font-style: normal;
   font-weight: 600;
   font-size: 13px;
   line-height: 16px;
   color: #7a869a;
 }
 
-#counter-Img {
-  background-image: url(images/Counter.png);
-  width: 8px;
-  height: 8px;
-  display: flex;
+.lang {
+  border-radius: 8px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: none;
+  cursor: pointer;
+  font-size: 12px;
+  padding: 4px 12px;
+  list-style-type: none;
+  margin-right: 8px;
 }
 
-.skill-Img li {
-  display: flex;
-  justify-content: space-between;
+.project {
+  color: #396df2;
+  border-radius: 8px;
+  border: 1px #6070ff solid;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  word-spacing: 3px;
+  margin-top: -4px;
+  padding: 8px 12px;
+  text-align: center;
+  background-color: #fff;
 }
 
-.frame-Img li {
-  display: flex;
-  justify-content: space-between;
-  margin-bottom: 20px;
-  height: 72px;
+.project:hover {
+  background-color: #535ca8;
+  border-color: #535ca8;
+  color: #fff;
 }
 
-.Tags ul li {
-  width: 60px;
-  margin-left: 20px;
+.project:active {
+  background-color: #4053fc;
+  color: #fff;
 }
```

Comment:
- [ ] I appreciate the effort you put Into placing the `:hover` and `:active` state properties in changing the background color but please make sure they're aligned with the given Figma design.
    - Figma design:
      ![firefox_7KIb3eS8J1](https://user-images.githubusercontent.com/104968236/230643815-0b520798-3ddf-4e49-903b-9e7e3b6543b6.png)
    - So when hovered over the button the background should change to: `#6070FF` and a subtle box shadow (you can check the design)
    - And when pressing the button the background should change to: `#2230D2`